tests/cpu/Platform_tests.cpp

```diff
@@ -93,13 +93,13 @@ OCIO_ADD_TEST(Platform, setenv)
     }
 #ifdef _WIN32
     {
-        SetEnvironmentVariable("MY_WINDOWS_DUMMY_ENV", "1");
+        SetEnvironmentVariable(L"MY_WINDOWS_DUMMY_ENV", L"1");
         std::string env;
         OCIO_CHECK_ASSERT(OCIO::Platform::Getenv("MY_WINDOWS_DUMMY_ENV", env));
```

Comment:
I think that the code should highlight that OpenColorIO only works with UTF-8 strings, and unit tests are examples on how to use a method, class, etc. Here the Windows API call uses the 'L' to mention that the strings are UTF-16 strings, so the Getenv() arguments must be prefix with 'u8' to highlight that they are UTF-8 strings.

Reply:
To be consistent, the lines above must also be updated to 'u8' strings.

Reply:
Another comment: Based on the code changes, I think that the two previous unit tests (i.e. `(Platform, envVariable)`and `(Platform, getenv)`) must also add some tests around Windows & UTF-16.